Lib/idlelib/help_about.py

```diff
@@ -21,6 +22,10 @@ def build_bits():
         return architecture()[0][:2]
 
 
+def callback(url):
```

Comment:
This function is rather redundant, since you just use a lambda later - it'd be more straightforward just to inline the `webbrowser` module call below.

Reply:
See later comment on the bind statement.